.github/workflows/changelog.yml

```diff
@@ -14,18 +14,13 @@ jobs:
     - uses: actions/checkout@master
       with:
         ref: 'main'
-    - name: Set up Ruby 2.7
+    - name: Set up Ruby
       uses: ruby/setup-ruby@v1
```

Comment:
This library has improved a bunch since I added it (Or I read the docs a bit more), it now reads the `.ruby-version` file to decide on the version to install, plus it auto-caches gems.